### PR TITLE
dd wizard docs for crash reporting

### PR DESCRIPTION
**Pull request by louiszawadzki**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)
- [ ] If this PR is auto-generated, please make sure also to manually update the code related to the change

### CODE REVIEW

File: docs/crash_reporting.md [hunk 1]
```diff
@@ -62,6 +62,14 @@ In order to make your application's size smaller, its code is minified when it i
 -   dSYMs for your iOS native code
 -   Proguard mapping files if you have enabled code obfuscation for your Android native code
 
+To do setup your project for sending the symbolication files automatically run `npx @datadog/react-native-wizard`.
```
Comment (hestonhoffman) [inline docs/crash_reporting.md:65]:
```suggestion
To set your project up to send the symbolication files automatically, run `npx @datadog/react-native-wizard`.
```

File: docs/crash_reporting.md [hunk 2]
```diff
@@ -62,6 +62,14 @@ In order to make your application's size smaller, its code is minified when it i
...
+
+See the wizard [official documentation][13] for options.
```
Comment (hestonhoffman) [inline docs/crash_reporting.md:67]:
```suggestion
```
This is an internal repo and there isn't much documentation there, so this link doesn't seem useful.

File: docs/crash_reporting.md [hunk 3]
```diff
@@ -337,8 +353,8 @@ To verify your React Native Crash Reporting and Error Tracking configuration, yo
 [5]: https://reactnative.dev/docs/signed-apk-android#enabling-proguard-to-reduce-the-size-of-the-apk-optional
 [6]: https://github.com/datadog/dd-sdk-android-gradle-plugin
 [7]: https://github.com/cwhenderson20/react-native-crash-tester
-[8]: https://docs.datadoghq.com/getting_started/site/
 [9]: https://fastlane.tools/
 [10]: https://appcenter.ms/
 [11]: https://www.bitrise.io/
 [12]: https://github.com/DataDog/datadog-ci/tree/master/src/commands/react-native#xcode
+[13]: https://github.com/DataDog/datadog-react-native-wizard
```
Comment (hestonhoffman) [inline docs/crash_reporting.md:360]:
```suggestion
```

**Review by hestonhoffman — COMMENTED** [review]
I noticed the wizard repo is internal, but [this doc is shared publicly](https://docs.datadoghq.com/real_user_monitoring/error_tracking/reactnative/#symbolicate-crash-reports). Will the steps work if the user doesn't have access to that repo? Or are there plans to make the repo public?

File: docs/crash_reporting.md (hunk 2 above)
Comment (xgouchet) [reply docs/crash_reporting.md:67]: @louiszawadzki do we plan on make the wizard open source?

**Review by xgouchet — APPROVED** [review]

**Comment by louiszawadzki** [comment]
Hi @hestonhoffman @xgouchet,

To clarify the wizard repo is going to be made public this week. I'm preparing the docs in advance, but this PR won't be merged before the wizard repo is public

**Comment by hestonhoffman** [comment]
@louiszawadzki Great, I'll take another quick look with that context

**Comment by louiszawadzki** [comment]
@hestonhoffman the wizard is now open source :)

**Review by cswatt — APPROVED** [review]

**Pull request merged**